content/learn/book/getting-started/resources/_index.md

```diff
@@ -45,7 +45,7 @@ fn greet_people(
     // if that caused the timer to finish, we say hello to everyone
     if timer.0.tick(time.delta()).just_finished() {
         for name in &query {
-            println!("hello {}!", name.0);
+            println!("hello {name}!");
```

Comment:
```suggestion
            println!("hello {name.0}!");
```
`Name` doesn't implement display so this wouldn't work, I think.

Reply:
Name does implement Display!

Reply:
@cart Are we talking about this `Name`: https://docs.rs/bevy/latest/bevy/core/struct.Name.html

Or the `Name` defined here: https://bevyengine.org/learn/book/getting-started/ecs/

In the context of the book, I think it's the later, and I don't see Display implemented on it?

Reply:
Ahhhh yeah you're right. I was validating code by pasting it into something that had `bevy::prelude::*`. Nice catch /  thanks for doubling down!